cli.js

```diff
@@ -62,6 +62,12 @@ function open(name) {
 			}
 
 			return openNpm(name);
+		}).catch(err => {
+			if (err.code === 'ENOTFOUND') {
+				console.error('No network connected detected!');
+			}
+
+			process.exit(1);
```

Comment:
This needs to be inside the above `if` statement and instead here you need to rethrow the error:

```js
throw err;
```

Otherwise we hide any other error that might be thrown.